agent/src/event_win.cc

```diff
@@ -128,6 +128,9 @@ void ContentAnalysisEventWin::Shutdown() {
     FlushFileBuffers(hPipe_);
     hPipe_ = INVALID_HANDLE_VALUE;
   }
+  if (request()->has_print_data()) {
+    CloseHandle(reinterpret_cast<HANDLE>(request()->print_data().handle()));
```

Comment:
I think there is a double-`CloseHandle()` possibility if the user has created a `ScopedPrintHandle` from this request.  See other comments for more.